inspired by qhull-2003.1-pkgconfig.patch, available from git://pkgs.fedoraproject.org/qhull
requires injecting correct values INSTALLDIR and VERSION
author: Kenneth Hoste (UGent)
diff -Naur qhull/qhull.pc qhull/qhull.pc
--- qhull/qhull.pc	1970-01-01 01:00:00.000000000 +0100
+++ qhull/qhull.pc	2012-07-08 08:04:07.795336862 +0200
@@ -0,0 +1,11 @@
+prefix=INSTALLDIR
+exec_prefix=${prefix}
+includedir=${prefix}/include
+libdir=${prefix}/lib
+
+Name: qhull
+Description: General dimension convex hull library
+Version: VERSION
+Requires:
+Libs: -L${libdir} -lqhull -lm
+Cflags: -I${includedir}
